src/main/java/de/hysky/skyblocker/config/SkyblockerConfig.java

```diff
@@ -1092,4 +1098,9 @@ public String toString() {
 			return I18n.translate("text.autoconfig.skyblocker.option.richPresence.info." + name());
 		}
 	}
+
+	public static class Garden {
+		@SerialEntry
+		public boolean dicerTitlePrevent = true;
+	}
```

Comment:
Can you put this right below the spider's den class, please? This makes everything easier to track and gives fewer merge conflicts.